apps/C/cr_memory_test.c

```diff
@@ -1,10 +1,10 @@
 #define _ASMLANGUAGE
+#include "../defines/rvc_defines.h"
```

Comment:
Great that you added the rvc_defines..
But why are there so many "defines"  in this file too?

Reply:
Taken care of.